python/src/trezorlib/cardano.py

```diff
@@ -108,16 +103,85 @@ def create_input(input) -> messages.CardanoTxInputType:
 
 
 def create_output(output) -> messages.CardanoTxOutputType:
-    if not output.get("amount") or not (output.get("address") or output.get("path")):
+    if not output.get("amount") or not (
+        output.get("address") or (output.get("addressType") is not None)
+    ):
         raise ValueError("The output is missing some fields")
 
-    if output.get("path"):
+    if output.get("addressType") is not None:
+        return _create_change_address_output(output)
```

Comment:
What if it *is* None? Shouldn't we cry foul?

Reply:
See comment above.